installation/cmd/run.sh

```diff
@@ -195,12 +196,35 @@ if [[ ${DUMP_DB} ]]; then
         sed -i 's/image\:.*compass-schema-migrator.*/image\: k3d-kyma-registry\:5001\/compass-schema-migrator\:'$DUMP_IMAGE_TAG'/' ${ROOT_PATH}/chart/compass/templates/update-expected-schema-version-job.yaml
     fi
 
+    REMOTE_VERSIONS=($(gsutil ls -R gs://sap-cp-cmp-dev-db-dump/ | grep -o -E '[0-9]+' | sed -e 's/^0\+//' | sort -r))
+    LOCAL_VERSIONS=($(ls migrations/director | grep -o -E '^[0-9]+' | sed -e 's/^0\+//' | sort -ru))
```

Comment:
I think we will have to pass the full path to the migrations dir because currently the `integration` [job](https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-incubator_compass/2456/pre-main-compass-integration/1542053141107707904) fails with (line 908) : 
`DB dump will be used to prepopulate installation
ls: cannot access 'migrations/director': No such file or directory`